plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/BaseTestTrinoS3FileSystemObjectStorage.java

```diff
@@ -403,11 +397,7 @@ public void testDeleteRecursivelyDirectoryWithObjectsAndDirectorySuffixPlacehold
 
                 assertTrue(fs.delete(new Path(directoryPath), true));
 
-                // TODO The directory with the key `directoryKey + "/dir4"` should have been deleted
-                assertThat(listPaths(fs.getS3Client(), getBucketName(), prefix, true))
-                        .containsOnly(
-                                directoryKey + "/",
-                                directoryKey + "/dir4/");
```

Comment:
Does this correspond to https://github.com/trinodb/trino/issues/13017?
if so, can you place "https://github.com/trinodb/trino/issues/13017" in the TODO line?